src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs

```diff
@@ -119,12 +116,6 @@ public override void ExecuteCommand()
             // The directory that contains msbuild
             packArgs.MsBuildDirectory = new Lazy<string>(() => MsBuildUtility.GetMsbuildDirectory(MSBuildVersion, Console));
 
-            if (Verbose)
-            {
-                Console.WriteWarning(LocalizedResourceManager.GetString("Option_VerboseDeprecated"));
-                Verbosity = Verbosity.Detailed;
```

Comment:
@spadapet `Command.Verbosity` is a common option in a base class. It should stay.
@toddm Consider removing `Option_VerboseDeprecated` string if it's not used anywhere else.